### PR TITLE
Add Related section

### LINKED ISSUE #1441: Searchable Awesome
Thought I'd share something I've been working on in small parts of my spare time. 

**Awesome Indexed**

https://awesome-indexed.mathew-davies.co.uk

**Why?**

I was looking for alternatives to a service, so wanted to find all mentions of it to look for something better. It can also be used for other purposes, ie. A search for "Symfony" will return podcasts, jobs, tools and so on ...

It's also different from the other "Search" projects in that it searches the content which has more value to me.

---

It's super simple, hopefully useful to others. It certainly was for me.

Enjoy. 

Ps. If this is the wrong place to post, feel free to remove. Also happy to hear suggestions.

### PR DESCRIPTION
**Pull request by ThePixelDeveloper**
Fixes #1441

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -607,6 +607,9 @@
 - [Parasite](https://github.com/ecohealthalliance/awesome-parasite#readme) - Parasites and host-pathogen interactions.
 - [Food](https://github.com/jzarca01/awesome-food#readme) - Food-related projects on GitHub.
 
+## Awesome Related Tools
+
+- [Awesome Indexed](https://awesome-indexed.mathew-davies.co.uk) - A way to search over the Awesome dataset. Great for discovery and finding alternative tools.
```
Comment (sindresorhus) [inline readme.md]: Try to concise the description down to one sentence.

File: readme.md [hunk 2]
```diff
@@ -607,6 +607,9 @@
 - [Parasite](https://github.com/ecohealthalliance/awesome-parasite#readme) - Parasites and host-pathogen interactions.
 - [Food](https://github.com/jzarca01/awesome-food#readme) - Food-related projects on GitHub.
 
+## Awesome Related Tools
```
Comment (sindresorhus) [inline readme.md]: This needs to be added to the TOC.

Comment (ThePixelDeveloper) [reply readme.md]: Done 👍

File: readme.md (hunk 1 above)
Comment (ThePixelDeveloper) [reply readme.md]: Done 👍

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1098952897721622529

**Pull request merged**